generator/support.go

```diff
@@ -671,7 +671,7 @@ func (a *appGenerator) makeCodegenApp() (GenApp, error) {
 
 	basePath := "/"
 	if sw.BasePath != "" {
-		basePath = "/"
+		basePath = basePath
```

Comment:
Sorry, between github's outage and something I was working on, I'm clearly not thinking straight.